pkgs/development/libraries/lcrq/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  stdenv,
+  fetchFromGitea,
+  lib,
+  ...
+}:
+stdenv.mkDerivation rec {
+  name = "lcrq";
+  version = "0.1.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "librecast";
+    repo = "lcrq";
+    rev = "v${version}";
+    sha256 = "sha256-s8+uTF6GQ76wG1zoAxqCaVT1J5Rd7vxPKX4zbQx6ro4=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.